packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,353 @@ Specific property types support a subset of [JSON Schema validations](https://to
 | `number`      | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with double-precision 64-bit IEEE 754 floating point number constraint | `minimum` and `maximum`     |
 | `integer`     | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with 32-bit signed two's complement integer constraint           | `minimum` and `maximum`     |
 | `array`       | [JSON Array](https://tools.ietf.org/html/rfc7159#section-5)                                                                         |                             |
+
+## Unique attribute
+
+<ApiLifecycle access="ea" />
+
+You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as an employee identification number. You can declare a maximum of five unique properties for each user type and five unique properties in the Okta group profile. Different user types can have the same or different unique properties (up to the limit of five per type).
```

Comment:
"the Okta user profile" -> "Okta user profile(s)".  Let's not have _our own documentation_ ignore multiple user types :)